Formula/heartbeat.rb

```diff
@@ -20,17 +21,18 @@ def install
 
     cd "src/github.com/elastic/beats/heartbeat" do
       system "make"
-      libexec.install "heartbeat"
+      (libexec/"bin").install "heartbeat"
+      libexec.install "_meta/kibana"
 
       (etc/"heartbeat").install Dir["heartbeat*.{json,yml}"]
       prefix.install_metafiles
     end
 
     (bin/"heartbeat").write <<~EOS
       #!/bin/sh
-        exec #{libexec}/heartbeat \
+        exec #{libexec}/bin/heartbeat \
         -path.config #{etc}/heartbeat \
-        -path.home #{prefix} \
+        -path.home #{prefix}/libexec \
```

Comment:
This could just be `libexec`, no? Unless I'm misunderstanding here.

Reply:
It is annoying that audit doesn't catch that.

Reply:
Agreed. We audit `pkgshare` stuff so IMO it'd be perfectly legitimate to audit this too.